app.js

```diff
@@ -0,0 +1,24 @@
+const express = require('express')
+
+const app = express()
+
+const { getTopics } = require('./controllers/topics.controller')
+
+const { 
+    handleCustomErrors, 
+    handlePsqlErrors, 
+    handleServerErrors
+    } = require('./errors/index.js')
+
+
+app.use(express.json())
```

Comment:
This isn't needed yet we are not dealing with any requests.

Reply:
Hi, I've actually removed these for now! I was struggling for ages to think how to check for the 404 path not found so was trying everything :/ . I nchelped and Katherine jogged my memory about the app.all and /*.